assembl/static2/js/app/routes.jsx

```diff
@@ -24,10 +28,6 @@ export default (
       <Route path=":slug/community" component={Community} />
       <Route path=":slug/terms" component={Terms} />
     </Route>
-    <Route path="/v2/">
-      <Route path=":slug/profile" component={Profile} />
-      <Route path=":slug/styleguide" component={Styleguide} />
-    </Route>
```

Comment:
Please let the style guide route outside of the App component:

<Router>
    <Route path="/v2/" component={App}>
        ........
     </Route>
     <Route path="/v2/">
        <Route path="styleguide" component={Styleguide} />
    </Route>
</Router>